src/lando/main/models/profile.py

```diff
@@ -1,15 +1,44 @@
 from __future__ import annotations
 
-from django.contrib.auth.models import User
+from django.contrib.auth.models import Permission, User
+from django.contrib.contenttypes.models import ContentType
 from django.db import models
 
 from lando.main.models.base import BaseModel
 
+SCM_PERMISSIONS = (
+    ("scm_conduit", "SCM_CONDUIT"),
+    ("scm_level_1", "SCM_LEVEL_1"),
+    ("scm_level_2", "SCM_LEVEL_2"),
+    ("scm_level_3", "SCM_LEVEL_3"),
+    ("scm_versioncontrol", "SCM_VERSIONCONTROL"),
+)
+
 
 class Profile(BaseModel):
     """A model to store additional information about users."""
 
+    class Meta:
+        permissions = SCM_PERMISSIONS
+
     user = models.OneToOneField(User, null=True, on_delete=models.SET_NULL)
 
     # User info fetched from SSO.
     userinfo = models.JSONField(default=dict, blank=True)
+
+    def update_permissions(self):
+        """Remove permissions (currently SCM) and re-add them based on userinfo."""
```

Comment:
```suggestion
        """Remove permissions and re-add them based on userinfo."""
```

Reply:
> 

The reason I state "currently SCM" is that other user permissions are not affected by this method at this time. I will reword it to say "Remove SCM permissions and re-add them based on userinfo" instead.